vts-libs/vts/heightmap.cpp

```diff
@@ -201,8 +201,8 @@ void Morphology<Operator>::run(int kernelRadius)
 {
     const std::size_t total(in_.rows * in_.cols);
 
-    UTILITY_OMP(parallel for)
-    for (std::size_t idx = 0; idx < total; ++idx) {
+    UTILITY_OMP(parallel for shared(kernelRadius))
```

Comment:
If kernel radius is not modidifier, shouldn't it be `firstprivate`?

Reply:
shared preserves the same meaning as before. i do not change program logic, just make it compile with msvc (this is a workaround for a bug in the compiler)